integrations/lib/testing/integration/integration_test.go

```diff
@@ -40,7 +40,7 @@ func (s *IntegrationSuite) TestVersion() {
 
 	versionMin, err := version.NewVersion("v11.0.0")
 	require.NoError(t, err)
-	versionMax, err := version.NewVersion("v13")
+	versionMax, err := version.NewVersion("v14")
 	require.NoError(t, err)
 
 	assert.True(t, s.Integration.Version().GreaterThanOrEqual(versionMin))
```

Comment:
I think we could even remove this because we will be always using the branch's version

Reply:
In theory this is still useful when running locally, as it will first default to whatever `teleport` you may have in your `PATH`.